csharp-refactoring/README.md

```diff
@@ -0,0 +1,7 @@
+## Refactoring in C#
```

Comment:
README.md should be committed separately than the project. So, just: 
- Remove the README.md from the project.
- Commit the changes. 
- And then create a separate commit adding the README.md

Reply:
@zanbor can you please remove the README.md completely from this PR. The team will add it later. 
Sorry, I was confused with what exactly should be done. 
This is the reason the build is failing. Thank you